### ISSUE TITLE
Exception in plugin on every project load in CLion

### ISSUE THREAD

**Issue by Yamakuzure** [turbanoff/RunConfigurationAsAction#11]
Whenever I open a project in CLion, I get an "Exception in plugin **Run Configuration as Action** (1.4.3) with the following stacktrace.

````
com.intellij.serviceContainer.AlreadyDisposedException: Cannot create com.jetbrains.cidr.cpp.cmake.workspace.CMakeWorkspace because container is already disposed: Project(name=readeasyi, containerState=DISPOSE_COMPLETED, componentStore=/home/sed/Development/readeasyi) (disposed)
	at com.intellij.serviceContainer.ComponentManagerImpl.throwContainerIsAlreadyDisposed(ComponentManagerImpl.kt:522)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:498)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:471)
	at com.jetbrains.cidr.cpp.cmake.workspace.CMakeWorkspace.getInstance(CMakeWorkspace.java:211)
	at com.jetbrains.cidr.cpp.execution.CMakeBuildConfigurationHelper.getTargets(CMakeBuildConfigurationHelper.java:99)
	at com.jetbrains.cidr.execution.CidrBuildConfigurationHelper.findTarget(CidrBuildConfigurationHelper.java:64)
	at com.jetbrains.cidr.cpp.execution.CMakeAppRunConfiguration.getCMakeTarget(CMakeAppRunConfiguration.java:98)
	at com.jetbrains.cidr.cpp.execution.CMakeAppRunConfiguration.getBuildProfiles(CMakeAppRunConfiguration.java:103)
	at com.jetbrains.cidr.cpp.execution.CMakeExecutionTargetProvider$getTargets$$inlined$runReadAction$1.compute(actions.kt:61)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:834)
	at com.jetbrains.cidr.cpp.execution.CMakeExecutionTargetProvider.getTargets(CMakeExecutionTargetProvider.kt:42)
	at com.intellij.execution.ExecutionTargetManagerImpl.lambda$getTargetsFor$1(ExecutionTargetManagerImpl.java:238)
	at com.intellij.execution.ExecutionTargetManagerImpl.doWithEachNonCompoundWithSpecifiedTarget(ExecutionTargetManagerImpl.java:278)
	at com.intellij.execution.ExecutionTargetManagerImpl.getTargetsFor(ExecutionTargetManagerImpl.java:236)
	at org.turbanov.actions.Bootstrap.getTargets(Bootstrap.java:164)
	at org.turbanov.actions.Bootstrap.removeForAllExecutorsImpl(Bootstrap.java:129)
	at org.turbanov.actions.Bootstrap.lambda$removeForAllExecutors$0(Bootstrap.java:123)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:216)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:27)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:195)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:213)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:184)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
````

The Reporting window says I shall report this to you, so here I am.

**Comment by turbanoff** [comment]
It's similar to #10 which is already fixed in version 1.4.4.
Please try latest version.

**Comment by Yamakuzure** [comment]
Perfect!
Thank you very much!
